install/pxe_boot_troubleshooting.md

```diff
@@ -8,30 +8,33 @@ This page is designed to cover various issues that arise when trying to PXE boot
     - [Aruba Configuration](#aruba-configuration)
     - [Mellanox Configuration](#mellanox-configuration)
   - [Next steps](#next-steps)
+    - [Node iPXE Retries and NIC Order](#pxe-nic-boot-order)
     - [Restart BSS](#restart-bss)
     - [Restart KEA](#restart-kea)
     - [Missing BSS Data](#missing-bss-data)
 
 In order for PXE booting to work successfully, the management network switches need to be configured correctly.
 
 <a name="#required-configuration"></a>
+
 ## Configuration required for PXE booting
 
 To successfully PXE boot nodes, the following is required:
 
 - The IP helper-address must be configured on VLAN 1,2,4,7. This will be where the layer 3 gateway exists (Spine or Leaf).
```

Comment:
```suggestion
- The IP helper-address must be configured on VLANs 1, 2, 4, and 7. This will be where the layer 3 gateway exists (Spine or Leaf).
```